src/EnergyPlus/HeatBalanceAirManager.hh

```diff
@@ -99,10 +99,12 @@ struct HeatBalanceAirMgrData : BaseGlobalStruct
     std::unordered_set<std::string> UniqueZoneNames;
     std::unordered_map<std::string, std::string> UniqueInfiltrationNames;
     bool ManageAirHeatBalanceGetInputFlag = true;
+    bool CalcExtraReportVarMyOneTimeFlag = true;
```

Comment:
This should be initialized to `false`. That explains why there are many diffs - it's always being calculated.

Reply:
It looks like existing oneTimeFlag patterns are initialized to true and set to false after the calculation is done, roughly like this.
```
oneTimeFlag = true
if (oneTimeFlag) {
    perform calculation
    oneTimeFlag = false
}
```
The many diffs might be due to the extra output variable? (rdd will be different)?